doc/fileformat/cable_cell.rst

```diff
@@ -0,0 +1,429 @@
+
+   This describes an *initial external concentration* object for ion ``ion`` with value ``val`` (unit [mM]]).
+
+.. label:: (ion-reversal-potential ion:string val:real)
+
+   This describes an *initial reversal potential* object for ion ``ion`` with value ``val`` (unit [mV]]).
+
+.. label:: (mechanism name:string [...(param:string val:real)])
+
+   This describes a (point or density) mechanism object of the mechanism called ``name``. This expression
+   accepts zero or more ``(param:string val:real)`` expressions. Each of these expressions sets the value of
+   parameter ``param`` to ``val``.
+   For example:
+
+   .. code:: lisp
+
+      (mechanism "hh" ("gl" 0.5) ("el" 2))
+
+   This expression creates an "hh" mechanism and sets the "gl" and "el" parameters of the mechanism to 0.5
+   and 2 respectively (units depend on the :ref:`nmodl <formatnmodl>` mechanism).
+
+.. label:: (ion-reversal-potential-method ion:string method:mechanism)
+
+   This creates a *reversal potential method* (able to modify the reversal potential) of ion ``ion`` from
+   mechanism ``method``.
+   For example:
+
+   .. code:: lisp
+
+      (ion-reversal-potential-method "ca" (mechanism "nersnt/ca"))
+
+.. label:: (current-clamp (envelope-pulse delay:real duration:real amplitude:real) freq:real)
+
+   This creates a *current clamp*. If the frequency ``freq`` (unit [Hz]) is zero, the current is a square
+   pulse with amplitude ``amplitude`` (unit [nA]) starting at ``delay`` (unit [ms]) and lasting for ``duration``
+   (unit [ms]). If ``freq`` is non-zero, the current is sinusoidal with amplitude ``amplitude`` and frequency
+   ``freq`` from time ``delay`` and lasting for ``duration``.
+   (More information about current clamps can be found :ref:`here <cablecell-stimuli>`).
+
+.. label:: (current-clamp [...(envelope time:real amplitude:real)] freq:real)
+
+   This creates a *current clamp* with an amplitude governed by the given envelopes (``time`` unit [ms] and
+   ``amplitude`` unit [nA]). A frequency ``freq`` (unit [Hz]) of zero implies that the generated current simply
+   follows the envelope. A non-zero ``freq`` implies the current is sinusoidal with that frequency and amplitude
+   that varies according to the envelope. (More information about current clamps can be found
+   :ref:`here <cablecell-stimuli>`).
+   For example:
+
+   .. code::
+
+      (current-clamp (envelope (0 10) (50 10) (50 0)) 40)
+
+   This expression describes a sinusoidal current with amplitude 10nA and frequency 40Hz and that lasts
+   from t = 0ms to t = 50ms, finally leaving the current at 0nA (final amplitude in the envelope).
+
+.. label:: (threshold-detector val:real).
+
+   This describes a *threshold-detector* object with value ``val`` (unit [mV]]).
+
+.. label:: (gap-junction-site)
+
+   This describes a *gap-junction-site*.
+
+*Paintable* and *placeable* properties and dynamics are placed on regions and locsets respectively.
```

Comment:
More precisely: region/locset expressions.

Reply:
They're placed on the locsets/regions generated from the expressions, not the expressions themselves. Is it unclear in the text?

Reply:
I see the sentence is correct, however, when using the terms as such, I'm inclined to think of them by the names I have given them in the label dict (I have region Region1 and locset Locset1). Which in turn might lead me to believe I can only address them with those labels. But, I can use expressions here, so I do not in fact need to explicitly label them.

"placed on regions and locsets respectively, generated from a given expression." ?

Reply:
I'm make the change

Reply:
:+1: